Citizens/templates.yml

```diff
@@ -30,6 +30,38 @@ robindamianwayne:
     uuid: 1dc8679a-3a38-43e2-807a-985598c54a95
     traitnames: spawned,type,inventory,owner,gravity,location
     name: Damian Wayne
+electro:
```

Comment:
There are two electros, Jamie Foxx and Francine Frye, which one is this? You will have to update